app/routes/app/t/$id.tsx

```diff
@@ -0,0 +1,376 @@
+import { Search16 } from "@carbon/icons-react";
+import {
+  Input,
+  Select,
+  Title,
+  Text,
+  Button,
+  Center,
+  Divider,
+  Tabs,
+  Paper,
+  Badge,
+  Avatar,
+  Checkbox,
+  Progress,
+} from "@mantine/core";
+import { Form, Link } from "@remix-run/react";
+import type { Submission, TopicWithMarketplace } from "~/domain";
+import { Detail } from "~/components/Detail";
+import * as Author from "~/components/Author";
+import { ProjectBadge } from "~/components/ProjectBadge";
+
+export default function Challenge() {
+  return (
+    <div className="mx-auto container mb-12">
+      <section className="flex flex-col md:flex-row space-y-7 md:space-y-0 space-x-0 md:space-x-5 py-12">
+        <main className="flex-1">
+          <div className="space-y-3">
+            <Title order={2}>Challenge Title</Title>
+            <div className="flex space-x-8">
+              <Detail>
+                <Detail.Title>Sponsor</Detail.Title>
+                <Author.Author />
+              </Detail>
+              <Detail>
+                <Detail.Title>Chain/Project</Detail.Title>
+                <div className="flex space-x-2">
+                  <ProjectBadge slug="solana" />
+                  <ProjectBadge slug="solana" />
+                </div>
+              </Detail>
+              <Detail>
+                <Detail.Title>Reward Pool</Detail.Title>
+                <Badge>100 SOL</Badge>
+              </Detail>
+              <Detail>
+                <Detail.Title>Submissions</Detail.Title>
+                <Badge>1000</Badge>
+              </Detail>
+              <Detail>
+                <Detail.Title>Reviews</Detail.Title>
+                <Badge>99</Badge>
+              </Detail>
+              <Detail>
+                <Detail.Title>Winners</Detail.Title>
+                <Badge>
+                  <Text size="sm" className="normal-case font-normal">
+                    Pending
+                  </Text>
+                </Badge>
+              </Detail>
+            </div>
+            <Text color="dimmed" className="max-w-2xl">
+              What’s the challenge What web3 topic do you want to crowdsource potential analytics questions for? Why?
+              What’s the challenge What web3 topic do you want to crowdsource potential analytics questions
+            </Text>
+          </div>
+        </main>
+        <aside className="md:w-1/5">
+          <Center className="space-x-5">
```

Comment:
These buttons are overflowing. Thinking we can either make the buttons smaller or change the layout to have a larger aside.